src/activity_actor.cpp

```diff
@@ -4077,6 +4078,127 @@ std::unique_ptr<activity_actor> haircut_activity_actor::deserialize( JsonIn & )
     return haircut_activity_actor().clone();
 }
 
+rummage_pocket_activity_actor::rummage_pocket_activity_actor( const item_location &iloc,
+        const action act_kind, const tripoint &tgt )
+    : kind( act_kind ), target( tgt )
+{
+    std::pair<item_location, int> it_loc = std::make_pair( iloc, iloc->count() );
+    item_loc.emplace_back( it_loc );
+}
+
+void rummage_pocket_activity_actor::start( player_activity &act, Character &who )
+{
+
+    int moves{};
```

Comment:
I think this initializes it to 0, but I only think, and am not sure - I do not think such confusion is necessary.
```suggestion
    int moves = 0;
```